### ISSUE TITLE
OC: AllowToggleSip support

### ISSUE THREAD

**Issue by stevezhengshiqi** [daliansky/XiaoMi-Pro-Hackintosh#574]
Currently, I don't have Xiaomi laptops again due to long vacation. https://github.com/acidanthera/OpenCorePkg/commit/eb5cc3ee62c06f111931a5e20ed584085ac28d1f adds a new quirk called `AllowToggleSip`, aiming to allow users to change SIP status in bootpicker. I have no idea about how does it work in detail, so I will disable this option for future releases until we have a clear idea about it.

Anyone who have interest on this is welcome to share their findings.

Basically I have two questions:
- Do we need to add csrutil.efi for this quirk?
- How do we change SIP in the bootpicker?

Ref:
https://github.com/acidanthera/OpenCorePkg/commit/eb5cc3ee62c06f111931a5e20ed584085ac28d1f
https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Configuration.pdf

**Comment by stevezhengshiqi** [comment]
Although I don’t have the machine, it seems pretty to use.
